presto-spi/src/main/java/io/prestosql/spi/security/ViewExpression.java

```diff
@@ -19,12 +19,12 @@
 
 public class ViewExpression
 {
-    private final String identity;
+    private final Identity identity;
```

Comment:
This class is used by connector and system access control. System level operates on Identity, while connector operates on ConnectorIdentity.

Reply:
Thanks! I now understand the difference between `Identity` and `ConnectorIdentity`.